scipy-1.0/paper.tex

```diff
@@ -567,14 +567,14 @@ \subsection*{SciPy Matures}
 
 By the middle to late 2000s, SciPy was starting to mature after a long phase of significant
 growth and adoption.
-The informal workshops are grown into international conferences with many
-hundreds of attendees, special issues are organized and published in a
-leading scientific magazine, and the scope of the SciPy library
-narrows, while the breadth of the ecosystem is grown and encouraged
+The informal workshops grew into international conferences with many
+hundreds of attendees. Special issues were organized and published in a
+leading scientific journal, and the scope of the SciPy library
+narrowed, while the breadth of the ecosystem grew
 through a new type of auxiliary package, the scikit.  The tooling,
-development, and release processes become more professional, grown with
-the patience affordable in open source development, leading with
-best practices nowadays increasingly common in both industry and other
+development, and release processes became more professional, grown with
+the patience affordable in open source development. Leading with
+best practices nowadays is increasingly common in both industry and other
 open source projects \cite{millman2014developing}.
 
 
```

Comment:
The next sentence also has a tense issue. It should be...
"In the early workshops, recurrent topics _indicated_..."
"reflected" might be a stronger word choice

Reply:
true -- fixed